setup.py

```diff
@@ -13,7 +13,6 @@
      author='Pankaj Kumar',
      author_email='pankaj.kmr1990@gmail.com',
      license='MIT',
-     py_modules=['bias_correction'],
```

Comment:
https://packaging.python.org/guides/distributing-packages-using-setuptools/#py-modules

Reply:
is needed when not having `bias_correction.__init__`